worldengine/cli/main.py

```diff
@@ -313,6 +314,14 @@ def main():
                              "If not provided, the default values will be used. \n" +
                             "[default = .059/.222/.493/.764/.927/.986/.998]",
                             metavar="#/#/#/#/#/#/#")
+    g_generate.add_argument('-gv', '--gamma-value', dest='gv', type=float,
+                        help="N = Gamma value for temperature/precipitation " +
+                             "gamma correction curve. [default = %(default)s]",
+                        metavar="N", default='1.25')
```

Comment:
Do you actively use the ```metavar```-name somewher?
(Also in line 324.)

As far as I understand (https://docs.python.org/3/library/argparse.html#metavar) you shouldn't make use of that parameter and the (much more useful) ```dest```-name will be used in case it is needed.